0004/README.md

```diff
@@ -0,0 +1,514 @@
+# Policy Architecture for Npm Package Provenance
+
+*   Proposer: Wietse Venema ([wietse@gmail.com](mailto:wietse@gmail.com))
+*   GitHub Issue: TODO
+*   Status: [DRAFT](../README.md#meaning-of-status-codes)
+*   Original link: [Google docs](https://docs.google.com/document/d/1ulft22Nv6OXFeVRIrVfH2bK4QcRyvv0RwDt2dhJltK4/edit?usp=sharing) (see there for reviewer comments and rejected ideas)
+
+## Summary
+
+This document proposes an architecture for policy evaluation over SLSA
```

Comment:
Maybe I'm missing some context, but I see terms like "policy evaluation" and "policy" used early in this document, but it's not clear to me what they mean exactly. Perhaps a definition at the first time of use may be a good idea?

Reply:
Would it help if I add some text along the lines of: "A provenance policy associates a package name with expectations for provenance attestation properties. A package (name, version) satisfies the provenance policy iff its provenance meets the expectations for that package name."

Reply:
That would go a long way, thanks, although I also do think it helps to add a small `Background` or `Architecture` section or something similar later on that goes deeper these definitions.

Reply:
I have added a terminology section before "Objective" that covers "provenance policy" and what "satisfying" a policy means". Elsewhere there already is text that covers expectations and attestation properties at a high level, but I'm not sure this document is the right place for deeper details of how policy evaluation is implemented.

Reply:
@trishankatdatadog Are you OK with the text as amended?